samples/system-test/products.v1p3beta1.test.js

```diff
@@ -57,32 +56,32 @@ async function getProductOrFalse(productPath) {
 
 describe(`products`, () => {
   before(tools.checkCredentials);
-
   before(async () => {
-    // Create a test product for each test
-    await productSearch.createProduct({
-      parent: productSearch.locationPath(
-        testProduct.projectId,
-        testProduct.location
-      ),
-      productId: testProduct.productId,
-      product: {
-        displayName: testProduct.productDisplayName,
-        productCategory: testProduct.productCategory,
-      },
-    });
-    testProduct.createdProductPaths.push(testProduct.productPath);
+    // Create a test product set for each test
+    try {
+      await productSearch.createProduct({
```

Comment:
Is this creating something that will be unique for every test session?  Many of these tests can run at the same time, so we need to be careful about using unique uuid identifiers for named test assets.

Reply:
Will update to make sure all ids are unique.

Reply:
Fixed